installer/Migration/V5_1_0/lang-string/general.yaml

```diff
@@ -149,3 +149,6 @@ langStrings:
   -
     value: 'Invalid'
     unitId: invalid
+  -
+    value: 'Or login with'
```

Comment:
is there a way to handle 
`<html> Or login with`
kind of source? may be parameterizing like `{{html}} Or login with`